bull/pdl-queue/consumer.js

```diff
@@ -83,11 +89,11 @@ function setHeaders(metadata, byteLength, title) {
   return headers;
 }
 
-async function uploadToIA(zip, metadata, byteLength, email, job) {
+async function uploadZipToIA(zip, metadata, byteLength, email, job) {
   const bucketTitle = metadata.IAIdentifier;
   const IAuri = `http://s3.us.archive.org/${bucketTitle}/${bucketTitle}_images.zip`;
   metadata = _.omit(metadata, "coverImage");
-  let headers = setHeaders(metadata, byteLength, metadata.title);
+  let headers = setHeaders(metadata, byteLength, metadata.title, zip);
```

Comment:
1. "zip" should be in string.
2. I think it's better you directly send the content type so that you don't need to do another check in the `setHeaders()` function.